### PR TITLE
`Player::RemoveItem` overloads

### PR DESCRIPTION
**Pull request by IRacle1**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1306,12 +1306,41 @@ public bool RemoveItem(Item item, bool destroy = true)
...
+        /// Removes a list of <see cref="Item"/>'s filtered on a predicate from the player's inventory.
+        /// </summary>
+        /// <param name="predicate">The condition to satisfy.</param>
+        /// <param name="destroy">Whether or not to destroy the items.</param>
+        /// <returns>Returns count of a successfully removed <see cref="Item"/>'s.</returns>
+        public int RemoveItem(Func<Item, bool> predicate, bool destroy = true)
+        {
+            List<Item> enumeratedItems = new(ItemsValue);
+            int count = 0;
+            foreach (var item in enumeratedItems)
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Player.cs:1319]: Don't use var.

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -1306,12 +1306,41 @@ public bool RemoveItem(Item item, bool destroy = true)
             return true;
         }
 
+        /// <summary>
+        /// Removes a list of <see cref="Item"/>'s filtered on a predicate from the player's inventory.
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Player.cs:1310]: It doesn't remove a "list" of items. Follow the existing docs' style.

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -1306,12 +1306,41 @@ public bool RemoveItem(Item item, bool destroy = true)
...
+        /// </summary>
+        /// <param name="predicate">The condition to satisfy.</param>
+        /// <param name="destroy">Whether or not to destroy the items.</param>
+        /// <returns>Returns count of a successfully removed <see cref="Item"/>'s.</returns>
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Player.cs:1314]: Grammar.

File: Exiled.API/Features/Player.cs [hunk 4]
```diff
@@ -1306,12 +1306,41 @@ public bool RemoveItem(Item item, bool destroy = true)
             return true;
         }
 
+        /// <summary>
+        /// Removes a list of <see cref="Item"/>'s filtered on a predicate from the player's inventory.
+        /// </summary>
+        /// <param name="predicate">The condition to satisfy.</param>
+        /// <param name="destroy">Whether or not to destroy the items.</param>
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Player.cs:1313]: Grammar.

File: Exiled.API/Features/Player.cs [hunk 5]
```diff
@@ -1306,12 +1306,41 @@ public bool RemoveItem(Item item, bool destroy = true)
...
+        /// <returns>Returns count of a successfully removed <see cref="Item"/>'s.</returns>
+        public int RemoveItem(Func<Item, bool> predicate, bool destroy = true)
+        {
+            List<Item> enumeratedItems = new(ItemsValue);
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Player.cs:1317]: And.. why?

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]
Fix docs and grammer by the way.

**Comment by IRacle1** [comment]
I resolved all that I could

**Review by NaoUnderscore — APPROVED** [review]
I'll do some modifications later. It's fine as it as, more less.

**Pull request merged**